README.md

```diff
@@ -5,81 +5,84 @@
 ### Why equity compensation matters
 
 Businesses use a variety of methods to attract and retain the best talent.
-Some employers offer big salaries and bonuses, hip offices, work-from-home time, generous
-vacation, and even fancy on-site meals.
-Of all these methods,
-[**equity compensation**](http://www.investopedia.com/terms/e/equity-compensation.asp), a kind of
-ownership in the company in exchange for work, is unique in its ability to align company
-and employee interests over a long period of time, and potential for wealth creation.
-The first ~3,000 employees of Facebook split roughly $23 billion,
-[according to The Financial Times](https://www.ft.com/content/d6599ae0-5738-11e1-869b-00144feabdc0).
-
-Equity compensation is also unique in its complexity, and includes various forms, such as
-restricted stock, stock options, and RSUs.
+Some employers offer big salaries and bonuses, hip offices, work-from-home time, or
```

Comment:
Notes (@RacheltheEditor you can help me enforce these):
- Think we should be very judicious about the first links in a guide as someone will inevitably click them. Moved a formal definition of equity compensation (and that link) later.
- This FT link is paywalled so downgraded it from an inline link to a parenthetical (have added a separate issue to track that)
- The intro needs more forward links throughout to upcoming sections (added a few, could be more)

Reply:
Forward links will be easy to add in after structure is stabilized. Excellent first point. To second point, paywalled content in parenthetical is definitely better and we have to remember to add 💰emoji to those, and use them at a minimum.